app/src/main/js/components/auth/login.js

```diff
@@ -44,7 +44,12 @@ class Login extends Component {
         <div className="auth_image" >
           <img alt="devrant" src="./res/images/devrant_sidebar.png" />
         </div>
-        <div className="auth_form" >
+        <div className="auth_form"
```

Comment:
Why not change it to a `form`? It makes sense
Otherwise, we should write a handler on both fields to check `e.which == 13` on `keypress`, which I don't recommend